Modix.Data/Repositories/MessageRepository.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Modix.Data.Models.Core;
+
+namespace Modix.Data.Repositories
+{
+    public interface IMessageRepository
+    {
+        Task<IReadOnlyCollection<MessageEntity>> GetRecentUserMessagesAsync(ulong guildId, ulong userId, TimeSpan timespan);
+
+        Task AddAsync(MessageEntity message);
+
+        Task RemoveAsync(ulong messageId);
```

Comment:
Should be `DeleteAsync` for consistency. (Yes I'm picky).

Reply:
Actually, strictly speaking, it should be `TryDeleteAsync()` and return `Task<bool>`.

Reply:
Consistency is good. Deletion is a bit weird for this repository because I can't think of an instance where we would only want to delete one row at a time - or really, anything at all (unless we're required to by Discord or something). Maybe remove this method entirely for now.

Reply:
Agreed, fixed in 31374be.